### ISSUE TITLE
Missing pictures from multi-picture messages

### ISSUE THREAD

**Issue by coppolab** [carderne/signal-export#17]
When multiple pictures are attached to the same message, only one of them is actually exported.
After a rough analysis, it seems to be due to the picture naming scheme since exported pictures are named exploiting a hour/minute/second template which is not sufficient in this case. Probably, multiple pics are just colliding into a single file name.

This is an example (.md) for a single exported message with multiple pics:
```
[2021-05-01 00:42] Me: Hello!!!!  ![2021-05-01_signal-2021-04-30-231219.jpeg](./media/2021-05-01_sign
al-2021-04-30-231219.jpeg)  ![2021-05-01_signal-2021-04-30-231219.jpeg](./media/2021-05-01_signal-2021-04-30-231219.jpeg)  ![2021-05-01_signal-2021-04-30-231
219.jpeg](./media/2021-05-01_signal-2021-04-30-231219.jpeg)  ![2021-05-01_signal-2021-04-30-231219.jpeg](./media/2021-05-01_signal-2021-04-30-231219.jpeg)  !
[2021-05-01_signal-2021-04-30-231219.jpeg](./media/2021-05-01_signal-2021-04-30-231219.jpeg)
```
This was found under MacOS.
Thanks for the excellent project, please go on.

**Comment by carderne** [comment]
Oh wow can't believe I didn't notice this!

This should be fixed now in 44b7cbe0cc43a10db941ce6c0a43034ba7ddc632. Just added an incrementing 00, 01, 02... to each additional attachment. Let me know if that works for you.

**Comment by coppolab** [comment]
Excellent, I confirm it is fixed, thanks for your fast resolution.
To my knowledge, this is now the first software actually doing the job, among several other attempts around.
